### PR TITLE
Update install.sh

### PR DESCRIPTION
**Pull request by DolphinChips**
Correct a few typos and fix a bunch of shellcheck warnings

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -1,37 +1,39 @@
...
+cp /sdcard/abm/temp/boot/stockboot.img-zImage "/sdcard/bootset/$2/zImage"
 
 # Copy rd
-cp /sdcard/abm/temp/boot/stockboot.img-ramdisk.gz /dcard/bootset/$2/inird.cpio.gz
+cp /sdcard/abm/temp/boot/stockboot.img-ramdisk.gz "/sdcard/bootset/$2/inird.cpio.gz"
 
-# Create entery
+# Create entry
 cmdline=$(cat /proc/cmdline)
 echo  "  default    Entry 01" >> /sdcard/bootset/lk2nd/lk2nd.conf
```
Comment (DolphinChips) [inline install.sh:33]: This block still can be improved, so do not merge yet.

**Comment by nift4** [comment]
you are GUI?

**Comment by nift4** [comment]
I fixed conflicts

**Comment by DolphinChips** [comment]
> you are GUI?

Nope, I'm CLI

**Comment by nift4** [comment]
xD

**Comment by nift4** [comment]
why draft?

**Comment by luka177** [comment]
LGTM thanks Gui/Cli or who you are.

**Pull request merged**